contracts/stabilizer/PegStabilityModule.sol

```diff
@@ -0,0 +1,276 @@
+pragma solidity ^0.8.4;
+
+import "./../token/Fei.sol";
+import "./../pcv/PCVDeposit.sol";
+import "./../utils/RateLimitedMinter.sol";
+import "./IPegStabilityModule.sol";
+import "./../refs/CoreRef.sol";
+import "./../refs/OracleRef.sol";
+import "../Constants.sol";
+import "@openzeppelin/contracts/security/ReentrancyGuard.sol";
+import "@openzeppelin/contracts/utils/math/SafeCast.sol";
+import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+
+abstract contract PegStabilityModule is IPegStabilityModule, CoreRef, RateLimitedMinter, OracleRef, ReentrancyGuard, PCVDeposit {
+    using Decimal for Decimal.D256;
+    using SafeCast for uint256;
+    using SafeERC20 for IERC20;
+
+    /// @notice the fee in basis points for selling asset into FEI
+    uint256 public override mintFeeBasisPoints;
+
+    /// @notice the fee in basis points for buying the asset for FEI
+    uint256 public override redeemFeeBasisPoints;
+
+    /// @notice the amount of reserves to be held for redemptions
+    uint256 public override reservesThreshold;
+
+    /// @notice the PCV deposit target
+    IPCVDeposit public override target;
+
+    /// @notice the token this PSM will exchange for FEI
+    /// This token will be set to address 0 if the bonding curve accepts eth
+    IERC20 public override token;
+
+    /// @notice FEI contract
+    IFei public FEI;
+
+
+    // ----------- Events -----------
+
+    /// @notice event emitted when a new mint fee is set
+    event MintFeeChanged(uint256 oldMintFee, uint256 newMintFee);
+
+    /// @notice event emitted when a new redeem fee is set
+    event RedeemFeeChanged(uint256 oldRedeemFee, uint256 newRedeemFee);
+
+    /// @notice event emitted when reservesThreshold is updated
+    event ReservesThresholdChanged(uint256 oldReservesThreshold, uint256 newReservesThreshold);
+
+    /// @notice event emitted when target is updated
+    event TargetChanged(IPCVDeposit oldTarget, IPCVDeposit newTarget);
```

Comment:
Naming conventions in the repo say "...Update" instead of "...Changed" for events

Reply:
Also they put events in interface only